packages/edit-post/src/components/layout/index.js

```diff
@@ -345,14 +345,15 @@ function Layout( { initialPost } ) {
 					)
 				}
 				actions={
-					<ActionsPanel
+					<PostLayoutActionsPanel
```

Comment:
Weird name for a component, why it's not just ActionsPanel?

Reply:
`ActionsPanel` felt to me too 'generic'. I can update though..

Reply:
It's not that important but I felt at least "Post" was not really useful and "Layout" is a bit misleading, it's not about layout here, it's about saving/publishing so not sure how to name it better :).